### ISSUE TITLE
Add support for zsh autocompletion

### ISSUE THREAD

**Issue by surajnarwade** [knative/client#130]
Since cobra also supports zsh autocomplation hence $SUBJECT

(I would like to work on this :see_no_evil: )

**Comment by maximilien** [comment]
PR #132

**Comment by surajnarwade** [comment]
I wanted to work but anyway thanks :)

**Comment by maximilien** [comment]
Was not clear @surajnarwade if you were just requesting ... and I too am a zsh user and that annoyed me too but never looked to see if Cobra supported zsh until you opened this issue. Best.

**Comment by surajnarwade** [comment]
@maximilien No worries :smile:

**Comment by maximilien** [comment]
@surajnarwade I think this is merged now so please verify that close if OK. Thx

**Comment by moficodes** [comment]
i added source < (kn completion --zsh) in my .zshrc but getting
```_arguments:comparguments:319: can only be called from completion function
_arguments:comparguments:319: can only be called from completion function```

Am i doing something wrong?

@maximilien

**Comment by maximilien** [comment]
Nope @moficodes. The generated code comes from the Cobra project. Can you provide what version of Zsh you are using and I'll try to find what is supported from the Cobra code.

**Comment by rhuss** [comment]
Can this issue be closed as it has been implemented with #132 ?

**Comment by maximilien** [comment]
IMO yup
